modules/k8s/Makefile.tanka

```diff
@@ -23,6 +23,12 @@ endif
 	@tk tool charts vendor
 endif
 
+## Check if there are any patch files in the charts/ directory and apply them to the relevant charts
+k8s/tanka/charts/patch:
+	@for f in charts/*.patch; do \
```

Comment:
Hm this is failing for me with
```
/bin/bash: line 2: charts/*.patch: No such file or directory
```

Reply:
Potentially `charts` does not exist as a dir either. I don't think we have any, but it is something that we would support currently.

Reply:
Fixed.